packages/babel-plugin-material-ui/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@material-ui/babel-plugin-material-ui",
+  "version": "1.0.0",
```

Comment:
v4?

Reply:
It's not really linked to the version of the packages, it works for all versions with the name `@material-ui/nameHere` and a `index.d.ts` file

Reply:
I've changed the version as suggested.

> We could add a peer dependency like styled-components

Would need one for styles, core, and lab then.